Cloudflare/gateway_network/ingest/parser.yml

```diff
@@ -14,6 +14,11 @@ stages:
       - set:
           "@timestamp": '{{json_event.message.Datetime}}'
           cloud.account.id: '{{json_event.message.AccountID}}'
+      - set:
+          destination.address: '{{json_event.message.DestinationIP}}'
+          destination.domain: '{{json_event.message.SNI}}'
+        filter: '{{json_event.message.SNI is not in [""]}}'
```

Comment:
Could you elaborate on why the destination.address shouldn't be set if the SNI is not present ?

Reply:
Ah, that was not my intention.

I think that destination.address will still be set automatically to the value of destination.ip if it's not explicitly set.

But I should add a test file for that case, and I'll also maybe change the code to always explicitely define destination.address.